apps/studio/src/features/editing-experience/components/form-builder/FormBuilder.tsx

```diff
@@ -76,7 +76,7 @@ export default function FormBuilder(): JSX.Element {
 
   const subSchema = getComponentSchema(component.type)
   const data = previewPageState[currActiveIdx]
-  const ajv = new Ajv({ strict: false })
+  const ajv = new Ajv({ strict: false, logger: false })
```

Comment:
comment/perf(out of scope): Might want to move this out of the component/memoize this since it does not rely on any props. Might be a heavy operation, though I don’t think so la haha.

Reply:
Updated in 572d8e6d22262160b5b8de972f8f729f6709658b